Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG PHP_VERSION
+ARG PHP_EXTENSIONS
+
+FROM php:${PHP_VERSION}-fpm
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:**  As recommended by `hadolint` (https://github.com/hadolint/hadolint) you should alias 
‏

```suggestion
FROM php:${PHP_VERSION}-fpm as base

FROM base
```

Reply:
I added it, but `hadolint` doesn't seem to require aliasing `FROM` in this case :thinking:

On their rule list, the closest rules I can find that could apply to this case are [DL3022](https://github.com/hadolint/hadolint/wiki/DL3022) and [DL3024](https://github.com/hadolint/hadolint/wiki/DL3024), but they're not a problem here..